Mathlib/NumberTheory/LSeries/HurwitzZetaOdd.lean

```diff
@@ -65,8 +65,6 @@ lemma jacobiTheta₂''_conj (z τ : ℂ) :
 /-- Restatement of `jacobiTheta₂'_add_left'`: the function `jacobiTheta₂''` is 1-periodic in `z`. -/
 lemma jacobiTheta₂''_add_left (z τ : ℂ) : jacobiTheta₂'' (z + 1) τ = jacobiTheta₂'' z τ := by
   simp only [jacobiTheta₂'', add_mul z 1, one_mul, jacobiTheta₂'_add_left', jacobiTheta₂_add_left']
-  generalize jacobiTheta₂ (z * τ) τ = J
-  generalize jacobiTheta₂' (z * τ) τ = J'
```

Comment:
I think this one might decrease infoview readability: they shorten a bit the expression in the infoview and make it apparent that this is about some computation with complex exponentials.

I think it’d better to stay on the safe side and keep the `generalize` here.